src/storage/in_mem_csv_copier/in_mem_arrow_rel_copier.cpp

```diff
@@ -366,86 +480,80 @@ static void putValueIntoLists(uint64_t propertyIdx,
     }
 }
 
-void InMemRelCSVCopier::putPropsOfLineIntoLists(InMemRelCSVCopier* copier,
-    vector<PageByteCursor>& inMemOverflowFileCursors, CSVReader& reader,
-    const vector<nodeID_t>& nodeIDs, const vector<uint64_t>& reversePos) {
+template<typename T>
+void InMemArrowRelCopier::putPropsOfLineIntoLists(InMemArrowRelCopier* copier,
+    vector<PageByteCursor>& inMemOverflowFileCursors, const vector<shared_ptr<T>>& batchColumns,
+    const vector<nodeID_t>& nodeIDs, const vector<uint64_t>& reversePos, int64_t blockOffset,
+    int64_t& colIndex, char delimiter) {
```

Comment:
This function has been reimplemented to take `CopyDescription` instead of taking the delimiter directly. It's not really a hack, `CopyDescription` is passed into the function as a parameter because `getArrowList` is static, so it cannot directly access the class member `CopyDescription`.